xc7/primitives/bram/bram.pb_type.xml

```diff
@@ -354,13 +359,20 @@ reason, BRAM tiles use a "fused sites" mode to accomidate the cross site logic.
        ZALMOST_EMPTY_OFFSET[12:0]=13'b1111111111111
        ZALMOST_FULL_OFFSET[12:0]=13'b1111111111111
       </meta>
+      <meta name="type">bel</meta>
```

Comment:
This shouldn't be needed?  The metadata should be coming from ramb18e1.pb_type.xml, yes?

Reply:
I'm fine with refering to the techmap and/or prjrxray issue for details